cmd/srcd/cmd/sql.go

```diff
@@ -67,21 +67,26 @@ var sqlCmd = &cobra.Command{
 			logrus.Fatalf("could not start gitbase: %v", err)
 		}
 
+		logrus.Info("waiting for connection to be ready")
```

Comment:
This message will appear always, right? Maybe this could use `logAfterTimeout` to log only if it's not immediately available, like we do a few lines above this.